02-documentation/developer-guides/building-a-nextjs-blog-with-squidex.md

```diff
@@ -0,0 +1,588 @@
+---
+description: >-
+  How to build a Next.js blog powered by content from Squidex CMS.
+---
+
+# Building a blog with Squidex and Next.js
+
+### Introduction
+
+This tutorial will guide you to create a blog application, set up its schema, and add mock content on Squidex, and create a Next.js blog based on the data stored in your CMS.
+
+## Prerequisites
+
+To complete this guide, you need:
+
+- A Squidex account. You can create one at https://cloud.squidex.io
+- Node.js and Yarn installed on your computer. Follow these guides to install [Node.js](https://phoenixnap.com/kb/install-node-js-npm-on-windows) and [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/#windows-stable) if you haven't already.
+
+## Step 1: Creating the app on Squidex
+
+In Squidex, an application is an isolated data store with schemas that define how its content is structured and API endpoints to query its content. You can visit [this page](https://docs.squidex.io/02-documentation/concepts/apps) of the Squidex docs to learn more about apps in Squidex.
+
+Start by logging into your Squidex account dashboard at https://cloud.squidex.io/. Click _New App_ to create an empty application. Enter a name you choose for the app in the space provided and click _Create_ to create the app. For the sake of this guide, we will assume the name of your Squidex application is `squidex-blog`.
+
+## Step 2: Creating the posts schema on the CMS
+
+Now that you have created the app on Squidex, you will proceed to create the schema for the blog application. This schema will define the structure of your content.
+
+Still on your Squidex account dashboard at https://cloud.squidex.io, click _squidex-blog_ (or whatever name you chose) on the dashboard to enter the dashboard for your newly created app.
+
+On the left panel, select _Schemas_ and click the Plus(_+_) to start creating the posts schema. Enter `posts` in the space provided under _Name_, select _Multiple contents_ since you will be having many posts, and click _Create_ to create the schema.
+
+You will be creating three fields under the posts schema: Title for the title of your posts, Slug to define the URL of your posts, and Content to contain the body of your blog posts.
+
+| Field name | Type   | Editor   |
+| ---------- | ------ | -------- |
+| Title      | String | Input    |
+| Slug       | String | Slug     |
+| Content    | String | Markdown |
+
```

Comment:
Instead of explaining so much a few screenshots would be much better.

Reply:
If you are a windows user: I use screenpressor for windows with red annotations: https://www.screenpresso.com/